config/install/atm.settings.yml

```diff
@@ -1,8 +1,13 @@
 register_url: https://www.adtechmedia.io/admin/accounts/signup
```

Comment:
It's impossible
Drupal load config/install/atm.settings.yml by pattern "{module_name}.settings.yml"